webpack.config.development.js

```diff
@@ -2,15 +2,18 @@ const webpack = require("webpack");
 const HtmlWebpackPlugin = require("html-webpack-plugin");
 const ExtractTextPlugin = require("extract-text-webpack-plugin");
 
-const port = process.env.PORT || 3000;
-
 module.exports = {
   entry: {
     // vendor: ["add here"],
-    app: ["react-hot-loader/patch", "./src/scripts/index.js"]
+    app: [
+      "react-hot-loader/patch",
```

Comment:
do I really need react-hot-loader/patch in addition to webpack-hot-middleware/client?